tools/rna_tools/locarna/macros.xml

```diff
@@ -232,6 +239,16 @@
                        'sequence length' (default: 0.0 = no effect)" />
     </xml>
     
+    <xml name="common_heuristic_parameters_sparse">
+        <param  name="prob_unpaired_in_loop_threshold" argument="prob-unpaired-in-loop-threshold" type="float" value="0.00005" 
+                min="0.0" max="0.2"
+                label="Minimal / cutoff threshold for prob-unpaired-in-loop" /> 
```

Comment:
see exparanap_in_loop_parameters, for a better label. It would be good to have the exparna-p and sparse in loop options as uniform as possible (unfortunately, there is a difference in the actual command line args '_' vs '-'. I will probably change this in some future release.)